packages/cfpb-layout/src/molecules/card.less

```diff
@@ -1,25 +1,38 @@
-.u-card-link-text() {
+// @c = default state.
+// @v = `:visited` state.
+// @h = `:hover` state.
+// @f = `:focus` state.
+// @a = `:active` state.
+.u-link-card__colors( @c, @v, @h, @f, @a ) {
```

Comment:
Would it make more sense to use `@d` for default?

Reply:
I was following the format of `u-link__colors-base`, which uses `@c` https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/utilities.less#L187

Probably stands for "current"